thirdeye/thirdeye-frontend/app/pods/manage/alert/route.js

```diff
@@ -0,0 +1,295 @@
+import Ember from 'ember';
+import fetch from 'fetch';
+import moment from 'moment';
+import { checkStatus, checkStatusPost } from 'thirdeye-frontend/helpers/utils';
+
+/**
```

Comment:
Can these be defined inside the route declaration, or is there a reason why these variables are defined outside? I might be missing something here.